### PR TITLE
more docs for version 1.2

### PR DESCRIPTION
**Pull request by mperrin**
- Updated text on index.rst, including list of contributors
- Update and edits to the detector effects page to add some more explanations, and fix header formatting
- Add new docs page about the setup_sim_to_match_data function. 
- Minor, reorder the list of pages to move later the optical budgets page so the new stuff can go before that. It's more relevant now in flight in any case.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -30,8 +30,10 @@ See :ref:`using_api`.
 
    This documentation is complemented by an `Jupyter Notebook format quickstart tutorial <http://nbviewer.jupyter.org/github/spacetelescope/webbpsf/blob/stable/notebooks/WebbPSF_tutorial.ipynb>`_. Downloading and running that notebook is a great way to get started using WebbPSF.
 
-.. note::
-   *Getting help:* For help using or installing webbpsf, you can contact the STScI JWST Help Desk at jwsthelp.stsci.edu. Note that WebbPSF is included in the Astroconda python distribution, as well as being installable via :ref:`standard Python packaging tools <installation>`.
+
+.. admonition:: Getting Help
+
+   For help using or installing webbpsf, you can contact the STScI JWST Help Desk at jwsthelp.stsci.edu. Note that WebbPSF is included in the Astroconda python distribution, as well as being installable via :ref:`standard Python packaging tools <installation>`.
```
Comment (marcio-melendez) [inline docs/index.rst]: I like to add, maybe in parenthesis:
..contact the STScI JWST Help Desk at jwsthelp.stsci.edu, category WebbPSF/JWST Telescope.
Not many people realize that webbpsf and the OTE have their own category in the help desk.

File: docs/index.rst [hunk 2]
```diff
@@ -19,7 +19,7 @@ WebbPSF is a Python package that computes simulated point spread functions (PSFs
 
 
 **Contributors:**
-WebbPSF has been developed by Marshall Perrin, Shannon Osborne, Robel Geda, Joseph Long, Justin Otor, Jarron Leisenring, Neil Zimmerman, Keira Brooks, and Anand Sivaramakrishnan, with contributions from Marcio Meléndez Hernandez, Alden Jurling, Lauren Chambers, Ewan Douglas, Charles Lajoie, Megan Sosey, and Kathryn St.Laurent. We also are grateful to the contributors of the astropy-helpers and stsci package templates.
```
Comment (marcio-melendez) [inline docs/index.rst:22]: can you remove Hernandez?

**Review by marcio-melendez — COMMENTED** [review]
This looks good, thanks for the input on the detector page. I only have a couple of really small comments.

File: docs/index.rst (hunk 2 above)
Comment (Skyhawk172) [reply docs/index.rst:22]: While you're at it, can you spell out my full name: Charles-Philippe Lajoie
Thanks!

**Review by marcio-melendez — APPROVED** [review]

**Pull request merged**